tests/test_async_example_dag.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
```

Comment:
Could you add a note in the PR description about why we needed to create a new job? This information can be valuable in the future if we want to understand some of our decisions.

Reply:
added a note